### PR TITLE
Fix(RJC-242): grant and revoke permission without resource

### PR DESCRIPTION
**Pull request by FilippoRezzonico**

### CODE REVIEW

File: standalone_apis.go [hunk 1]
```diff
@@ -250,6 +255,11 @@ func buildQuery(resourceType string, resourceIDs []string, subjects []string, gr
 		queryPartForSubjectOrGroups["$or"] = append(tempQuery, groupsQuery)
 	}
 
+	if resourceType == "" {
+		query := queryPartForSubjectOrGroups
+		return json.Marshal(query)
```
Comment (fredmaggiowski) [inline standalone_apis.go:260]: no need for the query definition above

```suggestion
		return json.Marshal(queryPartForSubjectOrGroups)
```

File: standalone_apis.go [hunk 2]
```diff
@@ -175,7 +176,9 @@ func grantHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if reqBody.ResourceID == "" {
+	resourceType := mux.Vars(r)["resourceType"]
+
```
Comment (fredmaggiowski) [inline standalone_apis.go:180]: this empty line is useless

File: standalone_apis_test.go [hunk 3]
```diff
@@ -441,28 +520,37 @@ func TestGrantHandler(t *testing.T) {
...
-			ResourceID: "my-resource",
+			Subjects:    []string{"a"},
+			Groups:      []string{"b"},
+			Permissions: []string{"c"},
+			Roles:       []string{"d"},
 		})
-		req, err := http.NewRequestWithContext(ctx, http.MethodPost, "/", bytes.NewBuffer(reqBody))
-		assert.NilError(t, err, "unexpcted error")
+
+		req := requestWithParams(
```
Comment (fredmaggiowski) [inline standalone_apis_test.go:544]: follow the same indentation of other tests please

**Review by fredmaggiowski — CHANGES_REQUESTED** [review]
some minor changes but the implementation is fine, nice job!

File: standalone_apis.go (hunk 1 above)
Comment (FilippoRezzonico) [reply standalone_apis.go]: removed useless query definition

File: standalone_apis.go [hunk 4]
```diff
@@ -57,7 +57,9 @@ func revokeHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	if len(reqBody.ResourceIDs) == 0 {
+	resourceType := mux.Vars(r)["resourceType"]
+
```
[the remark this answers is not in the document]
Comment (FilippoRezzonico) [reply standalone_apis.go]: removed useless empty line

File: standalone_apis_test.go (hunk 3 above)
Comment (FilippoRezzonico) [reply standalone_apis_test.go]: indentation fixed

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**